src/transformers/pipelines/automatic_speech_recognition.py

```diff
@@ -18,7 +18,7 @@
 
 from ..file_utils import is_torch_available
 from ..utils import logging
-from .base import Pipeline
+from .base import ChunkPipeline
```

Comment:
Did this already exist before?

Reply:
What do you mean ? This was added in https://github.com/huggingface/transformers/pull/14225

Reply:
Thanks for the reference! All good then :-)